package.json

```diff
@@ -30,14 +30,14 @@
   },
   "peerDependencies": {
     "babel-eslint": "10.x",
```

Comment:
Could we please change this to use @babel/eslint-parser?

Context: https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint